### PR TITLE
Support jwt

### PR DESCRIPTION
**Pull request by bachlx-bitmark**
Update code for jwt

### CODE REVIEW

File: server/src/utils/index.js [hunk 1]
```diff
@@ -34,8 +39,26 @@ const verifySignature = (messageString, signatureHex, bitmarkAccountNumber) => {
...
 
+const validJWT = (token) => {
+  return new Promise((resolve) => {
+    jwt.verify(token, certJWT, {
+      algorithms: ['RS256'],
+    }, (error, decoded) => {
+      if (error) {
+        return resolve();
+      }
+      if (decoded && (decoded.iat * 1000) < Date.now() && (decoded.exp * 1000) > Date.now()) {
```
Comment (phieudu241) [inline server/src/utils/index.js:52]: Should add a comment here to explain what is business logic here

File: server/src/utils/index.js [hunk 2]
```diff
@@ -34,8 +39,26 @@ const verifySignature = (messageString, signatureHex, bitmarkAccountNumber) => {
...
+      }
+    });
+  });
+};
+
 
 module.exports = {
   responseError, newError,
   verifySignature,
+  validJWT,
```
Comment (phieudu241) [inline server/src/utils/index.js]: Should change method name from **validJWT** to **validateJWT**

File: server/src/utils/index.js (hunk 1 above)
Comment (bachlx-bitmark) [reply server/src/utils/index.js:52]: done

File: server/src/utils/index.js (hunk 2 above)
Comment (bachlx-bitmark) [reply server/src/utils/index.js]: done

**Review by phieudu241 — APPROVED** [review]

**Pull request merged**